### PR TITLE
DM-1331: documentation of triggerManualReconnectOnConnectionDrop

### PR DESCRIPTION
**Pull request by sourajitsaha17**
This needs to be cherry-picked to 10.15 and 10.14
*** the main task is not yet merged to 10.14 but for 'develop' and 10.15 it's already merged

### CODE REVIEW

File: content/protocol-integration/opcua-bundle/gateway-register.md [hunk 1]
```diff
@@ -239,7 +239,13 @@ gateway:
     # if they fail due to error code Bad_NodeIdUnknown. It assumes that the NodeIds are correct, but it hasn't been added to the
     # server's address space yet. The default value is false.
     recreateFailedItems: false
-
+  
+  # Server connectivity configuration
+  connectivity:
+    # If the autoReconnect in server's client configuration is set to false, then the gateway tries to reconnect manually.
```
Comment (BeateRixen) [inline content/protocol-integration/opcua-bundle/gateway-register.md:245]:
```suggestion
    # If the autoReconnect in the server's client configuration is set to false, then the gateway tries to reconnect manually.
```

**Review by Oezge-Akin — APPROVED** [review]

**Review by bggov — APPROVED** [review]

**Review by sag-eweingaertner — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@sourajitsaha17 Should I merge this to develop and 10.15 already? Or better wait until it can also be merged to 10.14?

**Comment by sourajitsaha17** [comment]
> @sourajitsaha17 Should I merge this to develop and 10.15 already? Or better wait until it can also be merged to 10.14?

Yes @BeateRixen you can merge it for develop and 10.15 because the main task is already merged for that.
I'll create another PR once we merge 10.14

**Comment by BeateRixen** [comment]
Merged to develop and cherry-picked to 10.15.

**Pull request merged**